web/styles/zulip.css

```diff
@@ -2354,14 +2354,35 @@ select.invite-as {
     }
 }
 
-#invite-method-choice {
-    margin-top: 2px;
+.invite_type_radio_section {
+    margin: 2px 0 2px 5px;
+    padding: 2px 0;
+
+    & label.radio {
+        display: inline;
+        margin: 5px;
+    }
+
+    & input {
+        float: left;
+        width: auto;
```

Comment:
I think this can be removed since default value for `width` property is anyways `auto` and I cannot see any other CSS setting it to something else.

Reply:
And it seems like some of the CSS here is duplicated from the settings CSS. I think we should deduplicate this by extracting the common CSS and define it on a new class.